Sources/PaymentButtons/PayPalCreditButton.swift

```diff
@@ -25,7 +25,7 @@ public final class PayPalCreditButton: PaymentButton {
     ///   - size: Size of the button. Default to standard if not provided.
     public convenience init(
         insets: NSDirectionalEdgeInsets? = nil,
-        color: Color = .white,
+        color: Color = .darkBlue,
```

Comment:
We should update the docstring above this init as well

Reply:
Missed that, thanks! I've updated the docstrings in `PayPalButton` and `PayPalPayLaterButton` as well